src/uint/div.rs

```diff
@@ -1,5 +1,7 @@
 //! [`Uint`] division operations.
 
+#![allow(trivial_numeric_casts)]
```

Comment:
Can it be more targeted?

Reply:
Moved to `Uint::const_div_rem` in [60fb612](https://github.com/RustCrypto/crypto-bigint/pull/376/commits/60fb612ca0446f80dc293d14d345cdb8f67cdbc3)